### DISCUSSION TITLE
Unity (game engine) compatibility?

### DISCUSSION THREAD

**Discussion by asfarley** [akkadotnet/akka.net#5751] [Q&A]
Hi,

Wondering if anyone has used Akka.NET recently for a Unity project. Is it compatible? 

I've found a couple of posts e.g. on reddit mentioning this but they're pretty outdated.

**Comment by Aaronontheweb** [comment]
I think the `ReceiveActor` can't work on Unity right now due to AOT restrictions, but `UntypedActor`s should work just fine - ccing @Arkatufus on our team who has some significant Unity experience. He can speak to it better than I can.
